thread-manager/examples/report.lua

```diff
@@ -0,0 +1,4 @@
+done = function(summary, latency, requests)
```

Comment:
I'll plead some ignorance here, so if I'm missing something important, feel free to tell me to buzz off :D but we don't use wrk/lua for benchmarking anywhere else in the system. Is it possible to use something like `criterion` instead? Idea just being to avoid yet another stack to maintain/understand

Reply:
well yes, bad plan. Criterion can not realistically be used here as bench takes 10 sec for 1 sample, but wrk + taskset is overkill. fixed with hyper client.

Reply:
are we switching to hyper client? Do we still need this lua?

Reply:
nope, leftover, good catch!